modules/service/src/test/scala/lucuma/odb/graphql/mutation/cloneObservation.scala

```diff
@@ -228,96 +222,96 @@ class cloneObservation extends OdbSuite {
 
   }
 
-  test("cloned observation should always be present/new/active") {
+  // test("cloned observation should always be present/new/active") {
```

Comment:
Do we want to keep this code? If so, maybe comment on why it is commented out. Same for the commented out tests in the next few files.

Reply:
Yeah I will go back over these this morning and see if they're still relevant.